### PR TITLE
UNITY-5970: Add multiple lineitems to the cart

### PR DESCRIPTION
**Pull request by mhoehn**

### CODE REVIEW

File: _raml/apps/carts/examples/post_carts_cartid_multilineitem_request.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[
+  {
+    "productId": "54E72E1F-746A-8B6E-8E41-0A0C05E64669",
+    "quantity": 1
+  },
+  {
+    "productId": "54E72E1F-746A-8B6E-8E41-0A0C05E62569",
+    "quantity": 1
```
Comment (fgehring) [inline _raml/apps/carts/examples/post_carts_cartid_multilineitem_request.json]: Please use different GUIDs and other quantities than 1 for a better example and probably 3 items.

File: _raml/apps/carts/carts.raml [hunk 2]
```diff
@@ -104,6 +104,21 @@ post:
               example: !include examples/post_carts_cartid_lineitem_response.json
               schema: !include schema/post_carts_cartid_lineitem.schema.json
 
+  /multi-line-items:
+    displayName: Multiple line items
+    post:
+      description: Create product line items in a cart. *epages6 *epagesNow
```
Comment (cgebken) [inline _raml/apps/carts/carts.raml]: Creates multiple product line items ...

File: _raml/apps/carts/carts.raml [hunk 3]
```diff
@@ -104,6 +104,21 @@ post:
...
+      body:
+        application/json:
+          example: !include examples/post_carts_cartid_multilineitem_request.json
+          schema: !include schema/post_carts_cartid_multilineitem_request.schema.json
+      responses:
+        201:
+          body:
+            application/json:
+              example: !include examples/post_carts_cartid_lineitem_response.json
```
Comment (cgebken) [inline _raml/apps/carts/carts.raml]: This response does not match to the request as three line items are mentioned there. The response should thus also show these three items and not only one...

File: _raml/apps/carts/carts.raml (hunk 2 above)
Comment (mhoehn) [reply _raml/apps/carts/carts.raml]: fixed

File: _raml/apps/carts/carts.raml (hunk 3 above)
Comment (mhoehn) [reply _raml/apps/carts/carts.raml]: ok

**Comment by mhoehn** [comment]
PR now on develop-docs

**Pull request closed without merging**